tox.ini

```diff
@@ -28,13 +28,28 @@ commands =
     git submodule update
     make html
 
-[testenv:cov]
+[testenv:cov1]
```

Comment:
Can you add comments here describing the separation between cov1 and cov2?

Reply:
Yeah, that's a fair point.  Those names aren't that great either.  Good call.

Reply:
Done.